Incompatible with pnpm

If I try to launch the [basic server example](https://www.apollographql.com/docs/apollo-server/getting-started.html), I get `Error: Cannot find module 'apollo-server-env'`:

```
$ pnpm init -y
$ pnpm install --save apollo-server graphql
$ node index.js
```

--- a/packages/graphql-extensions/package.json
+++ b/packages/graphql-extensions/package.json
@@ -18,7 +18,7 @@
     "apollo-server-env": "file:../apollo-server-env"
   },
   "devDependencies": {
-    "apollo-server-core": "file:../apollo-server-core",
+    "apollo-server-core": "file:../apollo-server-core"
   },
   "peerDependencies": {
     "graphql": "^0.12.0 || ^0.13.0 || ^14.0.0"

--- a/packages/graphql-extensions/package.json
+++ b/packages/graphql-extensions/package.json
@@ -14,11 +14,11 @@
     "node": ">=6.0"
   },
   "dependencies": {
-    "@apollographql/apollo-tools": "^0.3.6"
+    "@apollographql/apollo-tools": "^0.3.6",
+    "apollo-server-env": "file:../apollo-server-env"
   },
   "devDependencies": {
     "apollo-server-core": "file:../apollo-server-core",
-    "apollo-server-env": "file:../apollo-server-env"
   },
   "peerDependencies": {
     "graphql": "^0.12.0 || ^0.13.0 || ^14.0.0"

